docs/reference.md

```diff
@@ -114,3 +114,64 @@ Message communication example
 |                                    | ← {"source":"custom_embed","action":"data","key":"j0a","data":{…}}                                        |
 |                                    | /or/ User cancelled changes                                                                   |
 |                                    | ← {"source":"custom_embed","action":"cancel","key":"j0a"}                                                 |
+
+# Request additional Story ANS
+Customers can obtain addition ANS data from Ellipsis via `postMessage`. Ellipsis/Composer will send back the data with the additional fields of:
+
+1. Headlines
+2. Subheadlines
+3. Taxonomy
+
+To acheive this, simply
+- Add an additional flag `isAnsRequired` set to `true` in your API where `postMessage()` is happening. E.g.
+
+```javascript
+const sendMessage = function(action, data) {
+  window.parent.postMessage(
+    JSON.stringify({
+      source: 'custom_embed',
+      action,
+      data,
+      key: parseQueryString()['k'],
+      isAnsRequired: false    // Change this to true
+    }),
+    '*'
+  )
+}
+```
+- Which will send out a message to Ellipsis/Composer that looks like:
+```javascript
+{source: "custom_embed", action: "ready", data: {…}, key: "1234567890", isAnsRequired: true}
+```
+- Ellipsis/Composer will postMessage back with additional Story ANS data:
+```javascript
+MessageEvent 
+{
+   "message":"message",
```

Comment:
message is too generic. Could u change it to something like ans_data? Also to update the code in composer repo.
In the future we may add other communication messages, so it's better to be more specific.

Reply:
It is done. Changing the message to `ans_data`.

https://github.com/WPMedia/arc-composer-ui/pull/19